src/main/java/de/tum/in/www1/artemis/web/rest/FileResource.java

```diff
@@ -311,6 +288,89 @@ private boolean validateTemporaryAccessToken(String temporaryAccessToken, String
         return true;
     }
 
+    /**
+     * Helper method which handles the file creation for both normal file uploads and for markdown
+     * @param file The file to be uplaoded
+     * @param keepFileName specifies if original file name should be kept
+     * @param markdown boolean which is set to true, when we are uploading a file within the markdown editor
+     * @return The path of the file
+     * @throws URISyntaxException if response path can't be converted into URI
+     */
+    @NotNull
+    private ResponseEntity<String> handleSaveFile(@RequestParam("file") MultipartFile file, @RequestParam("keepFileName") Boolean keepFileName, boolean markdown)
```

Comment:
```suggestion
    private ResponseEntity<String> handleSaveFile(@RequestParam("file") MultipartFile file, @RequestParam("keepFileName") boolean keepFileName, boolean markdown)
```

Reply:
I've seen Boolean used quite often when it is passed as a request param. But since I am not an expert in this field, I can also change it to boolean.